ui/raidboss/data/05-shb/alliance/the_tower_at_paradigms_breach.ts

```diff
@@ -402,6 +401,40 @@ const triggerSet: TriggerSet<Data> = {
       netRegexKo: NetRegexes.startsUsing({ id: '5C7[34]', source: ['헨젤', '그레텔'], capture: false }),
       response: Responses.aoe(),
     },
+    {
+      id: 'Paradigm Hansel/Gretel Bloody Sweep',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '5C5[4567]', source: ['Hansel', 'Gretel'] }),
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime) - 5,
+      durationSeconds: 5,
+      suppressSeconds: 1,
+      alertText: (_data, matches, output) => {
+        if (matches.id === '5C54' || matches.id === '5C55') {
+          // Hansel is West and Gretel is East
+          if (parseFloat(matches.castTime) > 12) {
```

Comment:
Also, in your comment you said the cast times were 12 and 8? I assume you mean something bigger than 12? Or, how does this work?

Reply:
yea, I can double check the values here real quick. Maybe its better to just use the average between the two.

Reply:
it.s 7.7 seconds and 12.7 seconds. I can use half way of 10 instead for clarity